client/src/panels/welcome_mat/welcome_mat.yaml

```diff
@@ -8,9 +8,9 @@ no_wrap_year:
   handlebars_partial : true
   text: <span class='text-nowrap'>({{year}})</span>
 years_ago:
-  transform: [handlebars]
+  transform: [handlebars, markdown]
   en: |
-    {{actual_hist_years_apart}} years ago ({{oldest_hist_spend_year}})
+    {{actual_hist_years_apart}} {{fmt_year_plural_text actual_hist_years_apart}} ago ({{oldest_hist_spend_year}})
   fr: |
     Il y a {{actual_hist_years_apart}} ans ({{oldest_hist_spend_year}})
```

Comment:
@cookkate or @AlexCLeduc, does the French also need a separate plural and singular case here?

Reply:
Yes, 'Il y a un an' or 'Il y a deux ans' would be correct. Note that the number is spelled out in words, see [here](http://dev.ea-ad.ca/welcome_matt_dead_programs/index-fra.html#orgs/program/TBC-BXA01/infograph/financial) (the "1 ans" vs "trois ans" in the headers)

Reply:
So I did "Il y a 1 an". Is that incorrect?